src/stream-analytics/azext_stream_analytics/_help.py

```diff
@@ -0,0 +1,370 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['stream-analytics job'] = """
```

Comment:
`-` in command name is not recommend practice right?

Reply:
There are many commands have `-`, especially in extensions. Can we use a abbreviation such as `sa` for it ? I prefer `stream-analytics`, because I think it is clearer. Anyone else can give your opinions ?